cli/cmd/install.go

```diff
@@ -207,6 +208,7 @@ func newCmdInstall() *cobra.Command {
 	// the configuration in validateAndBuild.
 	flags := options.recordableFlagSet()
 	installOnlyFlags := options.installOnlyFlagSet()
+	disableIdentityFlag := options.proxyConfigOptions.disableIdentityFlagSet()
```

Comment:
you could just do `options.disableIdentityFlagSet()` (though i could see reasons for making the convention either way)

Reply:
`options` is of type `*installOptions` in `install.go`.
`disableIdentityFlagSet()` is also called from `inject.go`:
https://github.com/linkerd/linkerd2/blob/ac6bd5eb6a2ca1e1539a949e53a8a8b0b507e830/cli/cmd/inject.go#L47
where `options` is of type `*proxyConfigOptions`. So I'm forced to use the most narrow common type.

Reply:
i may be misunderstanding, since `installOptions` has an in-lined `*proxyConfigOptions` member, you should be able to just call `options.disableIdentityFlagSet()` as-is, without specifying `proxyConfigOptions`?

Reply:
You're actually right :grin: